dask/multiprocessing.py

```diff
@@ -29,9 +47,11 @@ def get(dsk, keys, optimizations=[], num_workers=None,
     num_workers: int
         Number of worker processes (defaults to number of cores)
     func_dumps: function
-        Function to use for function serialization (defaults to dill.dumps)
+        Function to use for function serialization (defaults to
+        cloudpickle.dumps)
     func_loads: function
-        Function to use for function deserialization (defaults to dill.loads)
+        Function to use for function deserialization (defaults to
+        cloudpickle.loads)
```

Comment:
Suggest putting the endline before the `(defaults` just to keep the full idea on a single line.